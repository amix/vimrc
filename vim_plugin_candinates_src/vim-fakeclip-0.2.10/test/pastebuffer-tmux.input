" Test: vim-fakeclip pastebuffer-tmux
runtime! plugin/fakeclip.vim
call vspec#hint({
\   'scope': 'fakeclip#_local_variables()',
\   'sid': 'fakeclip#_sid_prefix()',
\ })
function s:describe__s_read_pastebuffer_tmux()  "{{{1
  " FIXME: NIY
endfunction




function s:describe__s_write_pastebuffer_tmux()  "{{{1
  " FIXME: NIY
endfunction




function s:describe__s_read_pastebuffer()  "{{{1
  " FIXME: NIY
endfunction




function s:describe__s_write_pastebuffer()  "{{{1
  " FIXME: NIY
endfunction




" __END__  "{{{1
" vim: filetype=vim foldmethod=marker
