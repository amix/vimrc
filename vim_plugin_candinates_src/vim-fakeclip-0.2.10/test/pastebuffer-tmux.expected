==== s_read_pastebuffer 

==== s_read_pastebuffer_tmux 

==== s_write_pastebuffer 

==== s_write_pastebuffer_tmux 



**** Result ****


0 examples, 0 failures
